src/js/control-bar/volume-control/check-volume-support.js

```diff
@@ -11,17 +44,10 @@
  * @private
  */
 const checkVolumeSupport = function(self, player) {
```

Comment:
I think it might be better to split this into two methods. One that just does volume checking and one that does mute checking rather than trying to make one method that does both.
Then `checkVolumeSupport` will say in use for the volume control and `checkMuteSupport` will be used in the mute toggle.

Reply:
Sounds reasonable, working on it.